crates/bevy_ecs/src/query/mod.rs

```diff
@@ -12,9 +12,24 @@ pub use state::*;
 
 #[cfg(test)]
 mod tests {
+    macro_rules! hash_set {
```

Comment:
While cool, I think this is overkill. Why not construct the HashSet from an array using FromIter?

Reply:
Whoops, I forgot about that.